IFC4x3/Properties/p/PowerCapacity_1mAfVHVQfDtBcb3$W2OBiX/Documentation.md

```diff
@@ -0,0 +1 @@
+Power capacity of the equipment
```

Comment:
is it power capacity (like Wh) or capacitance (Farad) as in the primary data type?

Reply:
Here it is a power capacity in Wh ! I changed the Primary data type to PowerMeasure.
0871f7b28802c8b6bf8a2b767b0356cb11e25edd

Reply:
see comment it is not PowerMeasure its EnergyMeasure for Wh.

power is a per time measure

Reply:
840d7b293d3051eca8050639884528e00dfc5d4e